### PR TITLE
documentation: minor edits

### PR DESCRIPTION
**Pull request by sirspinach**

### CODE REVIEW

**Comment by secant** [comment]
Can you also fix the broken link to sparkPI.spec in README.md? 😃

**Comment by sirspinach** [comment]
got it

File: gettingStarted.md [hunk 1]
```diff
@@ -7,13 +7,13 @@ Quilt supports Go version 1.5 or later.
 
 Find Go using your package manager or on the [Golang website] (https://golang.org/doc/install).
 
-### Setup $GOPATH
+### Setup GOPATH
```
Comment (secant) [inline gettingStarted.md]: Set up

**Comment by secant** [comment]
Also, sparkPI.spec needs a missing quote on line 28 when defining the githubKey (sorry... 😬)

**Comment by sirspinach** [comment]
hmm? I don't see any missing quote on line 29?

**Comment by secant** [comment]
Whoops sorry, I must have backspaced on my text editor 😦. Never mind heh

**Comment by sirspinach** [comment]
"merged" by commit 84babfab.

**Pull request closed without merging**